.github/workflows/tests.yml

```diff
@@ -6,28 +6,29 @@ name: dbt Tests
 on:
   # Triggers the workflow on push or pull request events and also adds a manual trigger
   push:
-    branches: 
-      - 'develop' 
-      - '*.latest' 
-      - 'releases/*'
-  pull_request_target:
-    branches: 
-      - 'develop' 
-      - '*.latest' 
-      - 'pr/*'
-      - 'releases/*'
+    branches:
+      - "develop"
+      - "*.latest"
+      - "releases/*"
+  pull_request:
```

Comment:
@leahwicz wanted to run this change by you. I was confused why tests weren't picking up my change, I realized the behavior for the `pull_request_target` event checks out the target branch for the checkout step. I think this is what we want, I'd like to confirm though!